internal/cli/deployment_destroy.go

```diff
@@ -75,10 +76,12 @@ func (c *DeploymentDestroyCommand) Run(args []string) int {
 					Deployment: deployment,
 				},
 			}); err != nil {
-				c.ui.Output("Error destroying the deployment: %s", err.Error(), terminal.WithErrorStyle())
-				return ErrSentinel
+				c.ui.Output("Error destroying deployment %d: %s", deployment.Sequence, err.Error(), terminal.WithErrorStyle())
 			}
 		}
+		if err != nil {
```

Comment:
This `err` is now the `err` from line 44 and not any error returned from `app.Destroy`, I don't believe that is what we want. 

If we plan to continue destroy attempts we probably need to collect the errors in a slice? I'm no longer sure what the intended behavior is

Reply:
@catsby I think it'd be best to remove the error check/return here. You're right that this is actually the `err` from line 44, and we already catch/return that earlier (as we should). 

I think collecting the errors in a slice is a good idea. After the loop is finished, we can check if there's > 0 errors in the slice, and if there are, return an error saying that one or more destroy-ments failed. The output message will already have warned the user of which ones failed so no need to parse through the slice for anything more specific.